### ISSUE TITLE
Odd behavior of new quest SN03

### ISSUE THREAD

**Issue by NicolaosGreene** [KC3Kai/KC3Kai#619]
Just a heads up that this new quest seems to be yet another one of those quests that the developers had share a counter with another quest. I say this because the objective of this quest is to sink 6 submarines and it had the 80% when I unlocked it and it automatically completed when I selected it. I had, however, earlier completed Bd8 which is also to sink 6 submarines so thus I suspect that the two share a counter. Though due to the nature of this I cannot go back to test this more.

**Comment by bamboo3250** [comment]
And the SN07 Attain 4 victories in PVP as well. I completed today PvP (and claimed all PvP quest rewards) before clearing E4. After I cleared E4 and completed SN05 then SN07 appeared with 80% indicator. Anybody know what that means? :dizzy_face:

**Comment by dragonjet** [comment]
wait, are we tracking event quests? xD

**Comment by NicolaosGreene** [comment]
I thought mentioning the behavior was good thing to do. I have no idea what quest things exactly are going to be in the next version so I was trying to preemptively provide warnings against problems that could happen.

**Comment by ReiFan49** [comment]
do we need to make a milestone counter,
and turn the quest tracker into `milestoneStart`, `milestoneSuspension`, and `milestoneProgress` ?
`milestoneStart` is the `milestone` counter before the quest activated **once** on that period
`milestoneSuspension` is the `milestone` delay where the `milestone` keep counting, but the quest is not tracked off
`milestoneProgress` is the `milestone` counter required to make it pass (marked as pass if `milestone >= progress - (start+suspension)`)

not sure if we need to compile every quest tracking issue into one issue.

**Comment by ethrundr** [comment]
similar behavior found is on the PvP quest after E-4
the counter shared with the daily PvP, as far as I remembered, since I just do my daily 3 + 5 PvPs without having the quest turned on, and when I do, it was completed right away.

at that moment, the daily 5 PvPs and weekly 20 PvPs quest still active and not yet completed, but the 3 PvPs is already completed. this happen after I cleared the morning shift of PvP battles.
